cfgov/jinja2/v1/owning-a-home/explore-rates/index.html

```diff
@@ -101,14 +101,17 @@ <h4>Explore rate options</h4>
                             <div id="credit-score-alert"
                                  class="result-alert credit-alert u-hidden"
                                  role="alert">
-                              <p class="alert">
-                                Many lenders do not accept borrowers with credit scores less than 620.
-                                Even if your score is low, you may still have options.
-                                <a href="https://www.consumerfinance.gov/mortgagehelp/">
-                                  Contact a housing counselor
-                                </a>
-                                to learn more.
-                              </p>
+                                {# NOTE: Text needs to appear on same line as
+                                   paragraph tag, otherwise there is a space
+                                   in front of the notification text. #}
```

Comment:
whaaaa! Since when does whitespace get rendered at the start of a paragraph?

Reply:
Weird huh? But see screenshot.

Reply:
I saw, that's such a strange rendering.